cluster/src/main/java/io/scalecube/cluster/ClusterMembership.java

```diff
@@ -418,29 +399,31 @@ private void processUpdates(List<ClusterMember> updates, boolean spreadGossip) {
     for (final ClusterMember member : updates) {
       LOGGER.debug("Member {} became {}", member.address(), member.status());
       switch (member.status()) {
-        case SUSPECTED:
+        case SUSPECTED: {
```

Comment:
@antonkharenko removed curly braces. take a look at code changes in this block again, I changed ``case TRUSTED:``  block.